pkg/virtctl/pause/pause.go

```diff
@@ -122,7 +121,13 @@ func (vc *VirtCommand) Run(cmd *cobra.Command, args []string) error {
 			}
 			err = virtClient.VirtualMachineInstance(namespace).Pause(vmiName)
 			if err != nil {
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+				if vm.Spec.Running != nil && !*vm.Spec.Running {
+					return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v. VirtualMachine %s is not set to running", vmiName, err, vmiName)
+				} else if vm.Spec.Running != nil && *vm.Spec.Running {
+					return fmt.Errorf("Error pausing VirutalMachineInstance %s: %v", vmiName, err)
+				}
+				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v, check if VirtualMachine is running", vmiName, err)
```

Comment:
Last but not least, when err != NotFound, probably something worse happened, and we can just keep the generic error message as before. The "check if VirtualMachine is running" advice probably does not help here.